scripts/release_ppa.sh

```diff
@@ -114,7 +114,7 @@ Priority: extra
 Maintainer: Christian (Buildserver key) <builds@ethereum.org>
 Build-Depends: ${Z3DEPENDENCY}debhelper (>= 9.0.0),
                cmake,
-               g++-4.8,
+               g++-8,
```

Comment:
I think we can only do this when we use static builds instead - otherwise we will still end up with a binary depending on the wrong (and incompatible) libstdc++.

Reply:
the libstdc++ comes with the ppa. The debian build scripts are clever enough to figure all that out.

Reply:
Ok, I wouldn't have expected that.